_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp

```diff
@@ -423,27 +418,34 @@ mfxStatus VAAPIEncoder::QueryEncodeCaps(ENCODE_CAPS & caps, mfxU8 codecProfileTy
 
     if ((attrs[idx_map[VAConfigAttribMaxPictureWidth]].value != VA_ATTRIB_NOT_SUPPORTED) &&
```

Comment:
Same check is done multiple times. Maybe better to make it inline function?

Reply:
Done